Code/Searchor/__init__.py

```diff
@@ -68,3 +68,24 @@ def SearchApple(query):
   query = query.replace(" ", "-")
   query_URL = "https://apple.com/search/" + query
   return query_URL
+
+def SearchTarget(query):
+  query = query.replace(" ", "+")
+  # Using F-Strings for the Learning factor. Later it's used widely in projects.
```

Comment:
Do we really have to use F-strings though? I wouldn't see the point in my opinion, as we're not using multiple variables and only are adding one string to the variable. Just my opinion on this case.

Reply:
We don't need to. I can remove them and commit the Changes for a new Review.